### ISSUE TITLE
[a11y][Violation][ARIA specification] Issues reported on Notebook UI using IBM Equal Access Accessibility Checker tool

### ISSUE THREAD

**Issue by nkn2022** [jupyter/notebook#7489]
## Description

I am running an accessibility test suite with Notebook UI using the **IBM Equal Access Accessibility Checker** tool on Chrome browser. The results look good overall, only a few things are missing to reach a first formal level of compliance. So kindly help look into these. 

Issues reported were violating **WCAG 2.2** requirements. 
Ref: https://www.ibm.com/able/requirements/checker-rule-sets

Full report (includes information on element locations where issue were found): 
[Accessibility_Report-7a73e10e-f8a4-4a06-a78d-5174a1309c5b.xlsx](https://github.com/user-attachments/files/17400350/Accessibility_Report-7a73e10e-f8a4-4a06-a78d-5174a1309c5b.xlsx)

List of violations categorized based on the checkpoints. 

# Category: `ARIA specification`
Ref: https://www.ibm.com/able/requirements/checker-rule-sets

****Violation of Rule:** aria_attribute_valid**
> List of issues with elements that violate this rule with a screenshots

> > **Issue 1** 
> > The ARIA attributes "aria-pressed" are not valid for the element <jp-button> with ARIA role "none"
> > **Element location:** 
> > `<jp-button minimal="" appearance="stealth"  current-value="" tabindex="-1" title="Enable Debugger" aria-pressed="false" aria-label="Enable Debugger" aria-disabled="false" class="jp-DebuggerBugButton jp-ToolbarButtonComponent">`
> > **Screenshot:**
> > ![Image](https://github.com/user-attachments/assets/c463a41a-74d3-41a6-a465-9d60d7e576cf)

> > **Issue 2**
> > The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"
> > **Element location:** 
> > `<div data-windowed-list-index="0" tabindex="0"  aria-label="Code Cell Content" class="lm-Widget jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-selected jp-mod-noOutputs jp-mod-active">`
> > **Screenshot:**
> > ![Image](https://github.com/user-attachments/assets/d3d26346-9fcd-4b73-9c62-702b28009599)
> > **Element location:**
> > `<div data-windowed-list-index="0" tabindex="0" aria-label="Code Cell Content with Output" class="lm-Widget jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-selected jp-mod-active">`
> > **Screenshot:**
> > ![Image](https://github.com/user-attachments/assets/83570730-9acb-467c-8823-476ae404fbd1)

## Reproduce

- Install Chrome extension for IBM accessibility checker tool. Ref: https://www.ibm.com/able/toolkit/verify/automated
- Open Notebook UI on Chrome browser. Add some cells with code.
- Run the tool and we can see the above `Violation`s reported.

Here is the screenshot of the UI scanned for the attached report:
![Image](https://github.com/user-attachments/assets/895eebec-51ad-43cd-93bb-99c4a8193b68)

## Expected behavior

No violations are reported on the scanning to reach a first formal level of compliance.

**Comment by nkn2022** [comment]
See an update after testing with latest version here: https://github.com/jupyterlab/jupyterlab/issues/16891#issuecomment-3519117352
I have tested on the jupyterlab IDE as well as stand alone Notebook UI. The issues are specific to Notebook UI. So I will keep this issue until these issues are resolved in the base Jupyterlab code and is integrated into Notebook.
